charts/scalardb/templates/deployment.yaml

```
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "scalardb.fullname" . }}
+  labels:
+    {{- include "scalardb.labels" . | nindent 4 }}
+spec:
+  minReadySeconds: 0
+  replicas: {{ .Values.scalardb.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "scalardb.selectorLabels" . | nindent 6 }}
+  {{- with .Values.scalardb.strategy }}
+  strategy:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        {{- include "scalardb.selectorLabels" . | nindent 8 }}
+    spec:
+      restartPolicy: Always
+      terminationGracePeriodSeconds: 60
+      {{- with .Values.scalardb.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.scalardb.podSecurityContext | nindent 8 }}
+
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.scalardb.securityContext | nindent 12 }}
+          image: "{{ .Values.scalardb.image.repository }}:{{ .Values.scalardb.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.scalardb.image.pullPolicy }}
+          ports:
+            - containerPort: {{ .Values.scalardb.server.port }}
+            - containerPort: {{ .Values.scalardb.server.prometheus_http_endpoint_port }}
```

I think we can fix these ports.
```suggestion
            - containerPort: 60051
            - containerPort: 8080
```
https://github.com/scalar-labs/helm-charts/pull/12/files#r683937706